core/testing.go

```diff
@@ -24,8 +25,15 @@ import (
 
 // MustGetNonce returns the next nonce for an actor at the given address or panics.
```

Comment:
this comment should probably be updated. as stated i would expect panic if the actor at the given address is not found

Reply:
I changed it to panic instead 😄